src/cli/menuItems/gpsCommands.cpp

```diff
@@ -0,0 +1,45 @@
+#include "Particle.h"
+
+#include "gpsCommands.hpp"
+#include "cli/conio.hpp"
+#include "consts.hpp"
+
+#include "location_service.h"
+
+void CLI_GPS() 
+{
+    LocationPoint point = {};
+    char ch;
+    
+    while(1) 
+    {
+        if(kbhit()) 
+        {
+            ch = getch();
+            if('ch' == q)
+            {
+                break
+            }
+        }
+
+        LocationService::instance().getLocation(point);
+        displayInfo(point);
+    }
+}
+
+
+void displayInfo(LocationPoint point)
+{
+    SF_OSAL_printf("Latitude %f" __NL__ , point.latitude);
+    SF_OSAL_printf("Longitude %f" __NL__, point.longitude);
+    SF_OSAL_printf("Altitude %f" __NL__, point.altitude);
+    SF_OSAL_printf("Speed %f" __NL__, point.speed);
+    SF_OSAL_printf("Heading %f" __NL__, point.heading);
+    SF_OSAL_printf("Accuracy (horizontal / vertical) (%f/%f)" __NL__, 
+                    point.horizontalAccuracy, 
+                    point.verticalAccuracy);
+    SF_OSAL_printf("Satellites in use %d" __NL__, point.satsInUse);
+    SF_OSAL_printf("Satellites in view" __NL__, point.satsInView);
+    SF_OSAL_printf("Locked: %d" __NL__, point.locked);
+}
```

Comment:
It would be nice to have this as tabular output.

So this might be:
```
SF_OSAL_printf("%11s %11s %6s %6s %6s %4s %4s %9s %9s %1s",
                "Latitude",
                "Longitude",
                "Alt",
                "Speed",
                "Hdg",
                "HDOP",
                "VDOP",
                "Sats Used",
                "Sats View",
                "Locked");
while (1)
{
    instance.getLocation(point);
    SF_OSAL_printf("%11.6f %11.6f %6.3f %6.3f %6.3f %4.2f %4.2f %9d %9d %6d\r", 
                    point.latitude,
                    point.longitude,
                    point.altitude,
                    point.speed,
                    point.heading,
                    point.horizontalAccuracy,
                    point.verticalAccuracy,
                    point.satsInUse,
                    point.satsInView,
                    point.locked
                    );
}
```